src/pages/Home/components/TeamBlock/TeamSlider/TeamSlider.tsx

```diff
@@ -0,0 +1,70 @@
+import { TeamPerson } from '@pages/Team/components/TeamPerson/TeamPerson';
+import React from 'react';
+import {
+  TABLET_POINT,
+  MOBILE_POINT,
+  MOBILE_SMALL_POINT,
+  TEAM_PERSONS
+} from '../../../../../constants';
+import { Swiper, SwiperRef, SwiperSlide } from 'swiper/react';
+import { Icon } from '@components/Icon/Icon';
+
+interface TeamSliderProps {}
+
+const TeamSlider: React.FC<TeamSliderProps> = () => {
+  const swiperRef = React.useRef<SwiperRef>(null);
+  const [slidePerView, setSlidePerView] = React.useState<number | 'auto'>(4);
+
+  const nextElementSwiper = () => {
+    if (!swiperRef.current) return false;
+    swiperRef.current?.swiper?.slideNext();
+  };
+  const prevElementSwiper = () => {
+    if (!swiperRef.current) return false;
+    swiperRef.current?.swiper?.slidePrev();
+  };
+
+  return (
+    <div className="team-slider">
+      <Icon
+        iconName="arrow-left"
+        className="team-slider_arrow-left"
+        onClick={prevElementSwiper}
+      />
+      <Swiper
+        spaceBetween={20} // should be 80, as in Figma layout
+        grabCursor
+        keyboard={{ enabled: true }}
+        slidesPerView={slidePerView}
+        ref={swiperRef}
+        breakpoints={{
```

Comment:
amazing usage of constants here) It will be comfortable if requirements are changed, good job